appium/lib/screen-object/accessors/element.rb

```diff
@@ -37,88 +34,179 @@ def driver
       end
 
       def click
-          element.click
+        element.click
       end
 
       def value
-          element.value
+        element.value
       end
 
       def exists?
+        driver.no_wait
         begin
-          element.displayed?
-        rescue
+          driver.set_wait if element.displayed?
+          true
+        rescue StandardError
           false
         end
       end
 
       def element
-        driver.find_element(:"#{locator[0]}",locator[1])
+        driver.find_element(:"#{locator[0]}", locator[1].to_s)
       end
 
       def elements
-        driver.find_elements(:"#{locator[0]}",locator[1])
+        driver.find_elements(:"#{locator[0]}", locator[1].to_s)
       end
 
       def element_attributes
         %w[name resource-id value text]
       end
 
+      def rect
```

Comment:
Can we have a more meaningful and redable method name? Just `rect` doesn't help explaining what it does. Maybe we can call it `get_position`